build/azure-pipelines.integration.yml

```diff
@@ -7,15 +7,9 @@ trigger:
     include:
       - refs/heads/main
 
-# Only test a PR when a smoke test or the runtime has changed
+# Only test a PR when an integration test or the runtime has changed
```

Comment:
Should this comment be removed or updated now that this currently only runs manually on PRs?